api/v1beta1/kafkacluster_types.go

```diff
@@ -138,6 +138,15 @@ type RollingUpgradeConfig struct {
 	// distinct broker replicas with either offline replicas or out of sync replicas and the number of alerts triggered by
 	// alerts with 'rollingupgrade'
 	FailureThreshold int `json:"failureThreshold"`
+
+	// ConcurrentBrokerRestartsAllowed controls how many brokers can be restarted in parallel during a rolling upgrade. If
+	// it is set to a value greater than 1, the operator will restart up to that amount of brokers in parallel, if the
+	// brokers are within the same AZ (as specified by "broker.rack" in broker read-only configs). Since using Kafka broker
+	// racks spreads out the replicas, we know that restarting multiple brokers in the same rack will not cause more than
+	// 1/Nth of the replicas of a topic-partition to be unavailable at the same time, where N is the number of racks used.
+	// This is a safe way to speed up the rolling upgrade.
+	// +optional
+	ConcurrentBrokerRestartsAllowed int `json:"parallelBrokerRestarts,omitempty"`
```

Comment:
nit:
```suggestion
	ParallelBrokerRestarts int `json:"parallelBrokerRestarts,omitempty"`
```

Reply:
Ah, good catch. I later renamed `ParallelBrokerRestarts` to `ConcurrentBrokerRestartsAllowed` and forgot to rename the json prop. I will rename the json prop to `concurrentBrokerRestartsAllowed` too (so the other way around), to avoid changing names everywhere and to keep the later name. Hope that is fine with you...

Reply:
Addressed in 7ed1df6ef4904c0e909da364d701b9031d014a9e